src/solids4FoamModels/solidModels/vertexCentredNonLinGeomTotalLagDisplacementSolid/vertexCentredNonLinGeomTotalLagDisplacementSolid.C

```diff
@@ -1259,16 +1277,13 @@ bool vertexCentredNonLinGeomTotalLagDisplacementSolid::evolve()
             //Obtain undeformed surface vector field
             surfaceVectorField SfUndef = dualMesh().Sf();
 
-            // Update gradDRef
-            surfaceTensorField gradDRef = dualGradDf_;
-
             // Calculate geometric stiffness field for dual mesh faces
             Foam::Field<Foam::RectangularMatrix<Foam::scalar>> geometricStiffness
             (
                 geometricStiffnessField
                 (
                     SfUndef,

```

Comment:
I suggest renaming `SfUndef` as `dualSfUndef` following the convention that all dual mesh quantities start with the prefix `dual`.